src/test/crimson/seastar_runner.h

```diff
@@ -71,16 +71,28 @@ struct SeastarRunner {
     auto ret = app.run(argc, argv, [this] {
       on_end.reset(new seastar::readable_eventfd);
       return seastar::now().then([this] {
-	begin_signaled = true;
-	[[maybe_unused]] auto r = ::eventfd_write(begin_fd.get(), APP_RUNNING);
-	assert(r == 0);
-	return seastar::now();
+// FIXME: The stall detector uses glibc backtrace function to
+// collect backtraces, this causes ASAN failures on ARM.
+// For now we just extend timeout duration to 10000h in order to
+// get the same effect as disabling the stall detector which is not provided by seastar.
+// the ticket about migrating to libunwind: https://github.com/scylladb/seastar/issues/1878
+// Will remove once the ticket fixed.
```

Comment:
Can you please create an issue in our tracker and paste it here as a comment?
`// See: https://tracker.ceph.com/issues/<>`

Reply:
Attached.